token_auth/views/register.py

```diff
@@ -0,0 +1,28 @@
+from django.contrib.auth import get_user_model
+from rest_framework import status
+from rest_framework.permissions import AllowAny
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from rest_framework_simplejwt.tokens import RefreshToken
+
+from token_auth.serializers.register import RegisterSerializer
+
+User = get_user_model()
+
+class RegisterView(APIView):
+
+    permission_classes = [AllowAny, ]
```

Comment:
`,` needed only when there is 1 element in Tuple and not List. List can have a

Reply:
ok